### ISSUE TITLE
Unable to query player data

### ISSUE THREAD

**Issue by MakiseKurisu** [leonismoe/genshin-stats#2]
After #1 fix the error in user script is now gone. However the website still cannot return any data, and only an empty toast was shown. Tested with latest Chrome and Firefox.

![image](https://user-images.githubusercontent.com/2321672/160543320-5888717f-ab1c-4a59-8d1b-60817172c3d3.png)

**Comment by leonismoe** [comment]
It's a confirmed bug in Tampermonkey 4.14, see https://github.com/Tampermonkey/tampermonkey/issues/1418.
I'll release a workaround soon.

---

这个 BUG 是 Tampermonkey 4.14 引起的，详见 https://github.com/Tampermonkey/tampermonkey/issues/1418

**Comment by leonismoe** [comment]
问题已被修复并部署，请确保在 Violentmonkey 或者 Tampermonkey 中更新对应的用户脚本到 1.2.1 版本。
您也可以访问 https://genshin-stats.leonis.dev/genshin-stats.user.js 手动触发更新。

---

Bug fixed, be sure to update the user script to 1.2.1 in Violentmonkey or Tampermonkey.
You can also visit https://genshin-stats.leonis.dev/genshin-stats.user.js to trigger the update.

**Comment by MakiseKurisu** [comment]
I'm still getting the same error. I replaced `json` field with the following code:
```
        json: function() {
          return this.text().then(function(value) {
              console.log("value");
              console.log(value);
              return unsafeWindow.JSON.parse(value);
          }, function(reason) {
              console.log("reject");
              console.log(reason);
          });
```
and got the below output in the console when I clicked 查询:
```
value
{"data":null,"message":"","retcode":10103}
```

Double checked that I'm logged in https://bbs.mihoyo.com/ys/ and the UID is valid (my own UID).
